action/github-actions.js

```diff
@@ -22,14 +22,14 @@ const escapeMessage = (value) => (
 const setOutput = (name, value) => {
   const filePath = process.env.GITHUB_OUTPUT;
   const githubCmd = `${name}=${value}${os.EOL}`;
-  fs.appendFileSync(filePath, githubCmd, { encoding: 'utf8' });
+  process.env.CI ? fs.appendFileSync(filePath, githubCmd, { encoding: 'utf8' }) : console.log(githubCmd);
```

Comment:
Maybe this is more direct?

```suggestion
  filePath ? fs.appendFileSync(filePath, githubCmd, { encoding: 'utf8' }) : console.log(githubCmd);
```

Reply:
Yup! Good call :-)